### DISCUSSION TITLE
Allowing wiki contributions via Pull-Requests

### DISCUSSION THREAD

**Discussion by PaulFreund** [SynthstromAudible/DelugeFirmware#57] [Ideas]
I noticed that the Wiki has it's own git address in GitHub now but unfortunately is not integrated in Pull requests.

I'd like to propose we add a copy of all the wiki files to /wiki in the main repository and use an approach similar to this https://nimblehq.co/blog/create-github-wiki-pull-request to synchronize them to the GitHub Wiki so the community can contribute via Pull Requests.

What do you think?

**Comment by litui** [comment]
Just my opinion, but pull requests tend to be an awkward mechanism for wiki maintenance. That would be more suited to a Jekyll site or something. For a wiki what you generally want is a wiki maintainer who corrects grave errors and reverts bad edits.
